DISCLAIMER.md

```diff
@@ -0,0 +1,27 @@
+# Disclaimer
+
+Simple Icons asks that users read this disclaimer fully before including an icon in their project.
+
+## Table of Contents
+
+* [Copyrights & Trademarks](#copyrights--trademarks)
+* [Guidelines](#guidelines)
+* [Removal of Brands](#removal-of-brands)
+
+## Copyrights & Trademarks
+
+The Simple Icons library contains a number of icons with specific requirements around their usage. Where this is the case, we aim to provide access to the proper guidelines through the `guidelines` entry in our JSON file & in our NPM package and through links on our [website](http://simpleicons.org/).
+
+We ask that users of our library read these guidelines fully and display the icons in accordance with them and, where applicable, with the necessary permissions.
+
+If a logo or icon includes a registered trademark or trademark symbol we follow the recommendations outline [in our Contribution Guidelines](https://github.com/simple-icons/simple-icons/blob/develop/CONTRIBUTING.md#icon-guidelines) to decide whether to include the symbol or not.
+
+~Simple Icons cannot be held responsible for any legal activity raised by a brand over our users' usage of their icon.~
+
+## Guidelines
+
+Please note that the addition of guidelines to our library is an ongoing project so the absence of a `guidelines` entry for a particular icon does not imply that guidelines for that icon do not exist. Similarly, all guidelines are subject to change and, as we rely largely on our community to help us keep our library updated, the guidelines included in our library for a particular icon may be outdated. Therefore we also ask that users of our library do their own due diligence in researching a brand's guidelines.
+
+## Removal of Brands
+
+Should a brand wish for their icon to be removed from the package, please submit an issue on the [GitHub repo](https://github.com/simple-icons/simple-icons/issues/new/choose) explaining your affiliation with the company, and reasons for removal. We generally remove icons that no longer [meet our criteria](https://github.com/simple-icons/simple-icons/blob/develop/CONTRIBUTING.md#requesting-an-icon) twice a year in our major releases - but can occasionally make exceptions for immediate removal of brands.
```

Comment:
First, using "repo" seems inappropriate in this document and would prefer "repository". Second, given one of the other suggestions, I would suggest we move the actual link for the repo to the end of the document as:

```suggestion
Should a brand wish for their icon to be removed from the package, please submit an issue on the [GitHub repository] explaining your affiliation with the company, and reasons for removal. We generally remove icons that no longer [meet our criteria](https://github.com/simple-icons/simple-icons/blob/develop/CONTRIBUTING.md#requesting-an-icon) twice a year in our major releases - but can occasionally make exceptions for immediate removal of brands.

[GitHub repository]: https://github.com/simple-icons/simple-icons/issues/new/choose
```